RAGchain/benchmark/dataset/msmarco.py

```diff
@@ -91,7 +91,6 @@ def ingest(self, retrievals: List[BaseRetrieval], db: BaseDB, ingest_size: Optio
         :param retrievals: The retrievals that you want to ingest.
         :param db: The db that you want to ingest.
         :param ingest_size: The number of data to ingest. If None, ingest all data.
-        If you want to use context_recall and context_precision metrics, you should ingest all data.
```

Comment:
You can use context_precision without ingesting all data~~
Please, keep this annotation only for context_recall